packages/flutter/lib/src/widgets/form.dart

```diff
@@ -615,11 +658,30 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
           if (_hasInteractedByUser.value) {
             _validate();
           }
+        case AutovalidateMode.onUnfocus:
         case AutovalidateMode.disabled:
           break;
       }
     }
+
     Form.maybeOf(context)?._register(this);
+
+    if (autovalidateMode == AutovalidateMode.onUnfocus || widget.autovalidateMode == AutovalidateMode.onUnfocus){
+      return Focus(
```

Comment:
I worry about adding a Focus widget here since the TextField also has one. Maybe this is a sign that FormField shouldn't be concerned with focus. Or that a bigger change is needed to make sure FormField's FocusNode and its child's FocusNode don't interfere with each other.

Reply:
I think we won't have any problems. The only problem I experienced was the interaction with FocusScope.

Reply:
@gspencergoog Any concern with adding a second Focus widget with its own FocusNode above EditableText, which has a Focus widget and separate FocusNode? It seems like you would be able to independently focus one or the other when really they should be the same.

Reply:
I wouldn't allow it to request focus or participate in traversal. (set `canRequestFocus` to false and `skipTraversal` to true). If it is above the `EditableText` in the focus tree, then it will still receive the focus change events.

Focus traversal will be weird if you leave it focusable: you'll have to hit tab twice to leave the field.